requirements.txt

```diff
@@ -1,6 +1,4 @@
 #external libraries
-tkinter
-# Python version requirement
-python_version >= "3.8"
+tkinter == 8.6.8
```

Comment:
- Do not write any comment in this file 
- it should be tkinter==8.6.8 (no spaces)